Update docs (#14)
datawowio/datawow-ruby: README/moderation_docs.md

# Moderation class

We're going to explain about how to and example of usage

# Table of Content
- [How to use](#how-to-use)
- [Create](#create)
- [All](#all)
- [Find by](#find-by)

## How to Use
There are 3 methods available in moderation class (create, all, find_by). Each
method return a response of type `Datawow::Response`, which has two attributes,
`body` and `code`. You can simply access response's body by calling `.body`

### Create
```ruby
Datawow.project_key = YOUR_PROJECT_KEY
params = {
  data: data,
  postback_url: postback_url,
  postback_method: postback_method,
  custom_id: custom_id,
  info: info,
  sandbox: sandbox
}
Datawow.moderation.create(params)
```

> Data could be text or image or else according to your project's input type

Reviewer: ```suggestion
`data` could be text or image's url depends on your project's input type
```

#### params

| Field        | Type           | Required  | Description |
| ------------- |:-------------:| :-----:| :-----|
| data | string | **Yes** | URL of image |
| postback_url| string| No | URL for answer callback once image has been checked |
| postback_method| string | No | Configuration HTTP method GET POST PUT PATCH |
| custom_id | string | No | Custom ID that used for search |
| info | json | No | Additional info for Document Verification |
| sandbox | boolean | No | Moderate data using sandbox credit |

#### result
```
response = Datawow.moderation.create({
  data: YOUR_DATA,
  custom_id: 'data-1'
})

print(response.body)
```

> If you could create moderation successfully, you will receive response like below:

Reviewer: ```suggestion
You will receive response like below, once you created moderation successfully.
```
